<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolgaWindow</class>
 <widget class="QMainWindow" name="VolgaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>881</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_vol_1">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QFrame" name="map_frame_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>691</width>
        <height>311</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="label_m_map_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>671</width>
         <height>291</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/images/images/map.png</pixmap>
       </property>
      </widget>
      <widget class="QRadioButton" name="indicatorMoscow_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="indicatorVolga_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="indicatorCaspian_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>230</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="indicatorNovosibirsk_3">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>160</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="indicatorEnd_3">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>190</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_9">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>10</y>
        <width>161</width>
        <height>511</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLCDNumber" name="lcdNumber_volga_odo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>410</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLCDNumber{color:rgb(230, 97, 0);}</string>
       </property>
       <property name="digitCount">
        <number>7</number>
       </property>
       <property name="mode">
        <enum>QLCDNumber::Mode::Dec</enum>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::SegmentStyle::Flat</enum>
       </property>
       <property name="value" stdset="0">
        <double>999.000000000000000</double>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber_volga_day">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLCDNumber{color:rgb(230, 97, 0);}</string>
       </property>
       <property name="digitCount">
        <number>7</number>
       </property>
       <property name="mode">
        <enum>QLCDNumber::Mode::Dec</enum>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::SegmentStyle::Flat</enum>
       </property>
       <property name="value" stdset="0">
        <double>999.000000000000000</double>
       </property>
      </widget>
      <widget class="QPushButton" name="button_vol_continue">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>470</y>
         <width>141</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Continue</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Day</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>390</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Odometer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>440</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>KM</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_vol_crew">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>141</width>
         <height>271</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Crew</string>
       </property>
      </widget>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_vol">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>330</y>
        <width>691</width>
        <height>192</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'IBM Plex Sans'; font-size:10pt;&quot;&gt;Volga.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'IBM Plex Sans'; font-size:10pt;&quot;&gt;We hit a barricade on the tracks. The brake system was punctured and we're stuck here until we can get it fixed. A church in the distance has sounded its bells and the drawbridge crossing the river is up. We'll have to check out the church first...&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'IBM Plex Sans'; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'IBM Plex Sans'; font-size:10pt;&quot;&gt;Map markers:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'IBM Plex Sans'; font-size:10pt;&quot;&gt;!   - Next objective&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'IBM Plex Sans'; font-size:10pt;&quot;&gt;H - safehouse&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'IBM Plex Sans'; font-size:10pt;&quot;&gt;?  - Curiosity&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_volga_talk">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>490</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Talk</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_vol_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QFrame" name="frame_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>341</width>
        <height>511</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLCDNumber" name="lcdNumber_volga_energy">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLCDNumber{color:rgb(230, 97, 0);}</string>
       </property>
       <property name="digitCount">
        <number>7</number>
       </property>
       <property name="mode">
        <enum>QLCDNumber::Mode::Dec</enum>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::SegmentStyle::Flat</enum>
       </property>
       <property name="value" stdset="0">
        <double>999.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>10</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Energy</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="text_volga_sub">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>321</width>
         <height>411</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_vol_map">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>511</width>
        <height>511</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="label_map_volga">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>491</width>
         <height>491</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/images/images/map_volga.png</pixmap>
       </property>
      </widget>
      <widget class="QPushButton" name="button_volga_church">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>170</y>
         <width>21</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>!</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_volga_crane">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>410</y>
         <width>21</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>!</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_volga_terminal">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>230</y>
         <width>21</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>!</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_volga_radDump">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>350</y>
         <width>21</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>!</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_volga_bridge">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>170</y>
         <width>21</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>!</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_volga_safehouse">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>280</y>
         <width>21</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>H</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_volga_banditTower">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>330</y>
         <width>21</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_volga_oilTank">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>400</y>
         <width>21</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_volga_banditCamp">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>230</y>
         <width>21</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
